src/experiments.js

```diff
@@ -616,3 +616,15 @@ window.Altis.Analytics.onReady( () => {
 // Fire a ready event once userland API has been exported.
 const readyEvent = new CustomEvent( 'altis.experiments.ready' );
 window.dispatchEvent( readyEvent );
+
+document.querySelectorAll( '.ab-test-xb-preview' ).forEach( function ( xb ) {
+	let templates = document.querySelectorAll( 'template[data-parent-id="' + xb.dataset.clientId + '"]' );
+	let tabCount = 1;
+	for ( let i = 0; i < templates.length; i++ ) {
+		document.querySelector( '.ab-test-xb-preview__tabs[data-client-tabs="' + xb.dataset.clientId + '"]' ).innerHTML += ( '<button data-tab="' + tabCount + '" class="variant">Variant ' + tabCount + '</button>' );
+		let innherHTML = templates[i].innerHTML;
+		document.querySelector( '.ab-test-xb-preview__content[data-client-content="' + xb.dataset.clientId + '"]' ).innerHTML += ( '<div class="variant-content">' + innherHTML + '</div>' );
```

Comment:
We already have the content we need in each `<template>` component, if you look further up in the file you can see that we swap the contents out using something like the following:

```js
const tabContent = xb.querySelector( '.ab-test-xb-preview__content' );

// then in the on click handler:
tabContent.innerHTML = ''; // empty the content
const variant = templates[i].content.cloneNode( true );
tabContent.appendChild( variant );
```

Note that `querySelector()` and `querySelectorAll()` are also methods of any `Element` reference so they can be scoped to specific elements rather than always querying from `document`: https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector